editor/block-mover/test/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { BlockMover } from '../';
+
+describe( 'BlockMover', () => {
+	describe( 'basic rendering', () => {
+		it( 'should render two IconButton components with the following props', () => {
+			const blockMover = shallow( <BlockMover /> );
+			expect( blockMover.hasClass( 'editor-block-mover' ) );
+
+			const moveUp = blockMover.childAt( 0 );
+			const moveDown = blockMover.childAt( 1 );
+			expect( moveUp.type().name ).to.equal( 'IconButton' );
+			expect( moveDown.type().name ).to.equal( 'IconButton' );
+			expect( moveUp.props() ).to.include( {
+				className: 'editor-block-mover__control',
+				onClick: undefined,
```

Comment:
It will show up `undefined` because of how I have the JSX and the way I am creating the Element for the test, i.e. with `isFirst` undefined. Should I add in failing tests for a different expected behavior or should I be matching what is already here?